pkgs/applications/version-management/git-and-tools/git-imerge/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "git-imerge-${version}";
+  version = "0.7.0";
+
+  src = fetchurl {
+    url = "https://github.com/mhagger/git-imerge/archive/0.7.0.tar.gz";
```

Comment:
It might be nice to re-use the `version` attribute in the URL to ensure it always points to the correct version.